src/components/guide/Guide.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useRef, useState } from "react";
+import { Tabs, TabsList } from "@radix-ui/react-tabs";
+import GuideTitle from "./GuideTitle";
+import PieIcon from "./PieIcon";
+import BuildIcon from "./BuildIcon";
+import DeployIcon from "./DeployIcon";
+import ScaleIcon from "./ScaleIcon";
+import Tab from "./Tab";
+import TabInfo from "./TabInfo";
+import TabInfoImg from "./TabInfoImg";
+import TabIcon from "./TabIcon";
+import TabDescription from "./TabDescription";
+import useIsVisible from "@site/src/hooks/UseIsVisible";
+
+const tabs = ["connect", "build", "deploy", "scale"];
+const tabTimer = 5000;
+
+export default function Guide() {
+  const [selectedTab, setSelectedTab] = useState("connect");
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

better due to not needing to change if tab naming is changed